docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -306,3 +288,120 @@ Le code ci-dessus génère ce qui suit sur la page :
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### Structure du dossier
+
+Utilisez le composant `<FileTree>` pour afficher la structure d'un répertoire avec des icônes de fichiers et des sous-répertoires repliables.
+
+Spécifiez la structure de vos fichiers et répertoires avec une [liste Markdown non ordonnée](https://www.markdownguide.org/basic-syntax/#unordered-lists) dans `<FileTree>`.
+Créez un sous-répertoire en utilisant une liste imbriquée ou ajoutez un `/` à la fin d'un élément de liste pour le rendre comme un répertoire sans contenu spécifique.
+
+La syntaxe suivante peut être utilisée pour personnaliser l'apparence de l'arborescence des fichiers :
+
+- Mettre en évidence un fichier ou un répertoire en mettant son nom en gras, par exemple `**README.md**`.
+- Ajouter un commentaire à un fichier ou à un répertoire en ajoutant du texte après le nom.
+- Ajouter des fichiers et des répertoires de remplacement en utilisant soit `...` soit `...` comme nom.
```

Comment:
```suggestion
- Ajoutez des fichiers et des répertoires fictifs en utilisant soit `...` soit `…` comme nom.
```

Any thoughts about using "fictifs" instead of "de remplacement"? The feature is not really replacing something but adding a virtual space where some files or folders could appear.